package.json

```diff
@@ -258,6 +258,6 @@
         "url": "https://github.com/Azure/vscode-bridge-to-kubernetes"
     },
     "extensionMetadata": {
-        "expectedCLIVersion": "1.0.20231127.3"
+        "expectedCLIVersion": "1.0.20240109.1"
```

Comment:
For wider folks only - First this PR will go out and then this release PR will need actioning : https://github.com/Azure/Bridge-To-Kubernetes/pull/349